app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,116 @@
+require './config/environment'
+require 'rack-flash'
+
+class TweetsController < ApplicationController 
+
+  use Rack::Flash
+  configure do
+    set :public_folder, 'public'
+    set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
+  end
+
+  # the tweets index page
+  # this appears to be the users homepage once they're signed in
+  # can grab a users specific tweets - a user has many tweets
+  get '/tweets' do 
+
+      if current_user
+        @user = current_user
```

Comment:
we don't need to set an instance variable to current_user. we have access to current_user in the view already.